src/components/hero/_macro.njk

```diff
@@ -48,7 +48,7 @@
                         <p class="ons-hero__text">{{ params.text | safe }}</p>
                     {% endif %}
                 </div>
-                {% if params.officialStatisticsBadge == true %}
+                {% if params.variants == "grey" and params.officialStatisticsBadge == true %}
```

Comment:
You can just add one if to check if its the grey variant around both these ifs. Would probably be cleaner